assets/src/room_ui.ts

```diff
@@ -26,6 +26,53 @@ type MediaStreams = {
   videoStream: MediaStream | null;
 };
 
+export function addAudioStatusChangedCallback(
+  callback: (status: boolean) => any
+) {
+  audioButton.addEventListener("click", () =>
+    callback(!getAudioButtonStatus())
+  );
+}
+
+export function addVideoStatusChangedCallback(
+  callback: (status: boolean) => any
+) {
+  videoButton.addEventListener("click", () =>
+    callback(!getVideoButtonCurrentStatus())
+  );
+}
+
+export function setMuteIndicatorVisible(peer_id: string, active: boolean) {
```

Comment:
I would consider treating microphone and camera in the same way and call these functions

`setMicIndicator` and `setCamIndicator`.

Current naming `setMuteIndicatorVisible` suggests that this function always set indicator to be visible which is not true.